test/e2e_node/image.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
+	"k8s.io/kubernetes/pkg/kubelet/dockertools"
+	kubepod "k8s.io/kubernetes/pkg/kubelet/pod"
+)
+
+type ConformanceImage struct {
+	Image   kubecontainer.ImageSpec
+	Runtime kubecontainer.Runtime
+}
+
+func NewConformanceImage(containerRuntime string, image string) (ci ConformanceImage, err error) {
+	ci.Image = kubecontainer.ImageSpec{Image: image}
+	if containerRuntime == "docker" {
+		ci.Runtime = dockerRuntime()
+		return ci, nil
+	}
+	return ci, fmt.Errorf("Unsupported runtime : %s.", containerRuntime)
+}
+
+func dockerRuntime() kubecontainer.Runtime {
+	dockerClient := dockertools.ConnectToDockerOrDie("")
+	pm := kubepod.NewBasicPodManager(nil)
+	dm := dockertools.NewDockerManager(
```

Comment:
@liangchenye Just FYI, for setting up rkt as runtime. We will need:
1. systemd must be running on the machine
2. rkt api service be running on the machine
3. pass the endpoint of the rkt api service

Given 1. this probably will not work on container vm, right? @yujuhong @Random-Liu